accumulator/starter/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"math/rand"
+
+	accumulator "github.com/temporalio/samples-go/accumulator"
+	"go.temporal.io/sdk/client"
+)
+
+var WorkflowIDPrefix = "accumulate"
+
+var TaskQueue = "accumulate_greetings";
+
+func main() {
+	// The client is a heavyweight object that should be created once per process.
+	c, err := client.Dial(client.Options{
+		HostPort: client.DefaultHostPort,
+	})
+	if err != nil {
+		log.Fatalln("Unable to create client", err)
+	}
+	defer c.Close()
+
+	// setup which tests to run
+    // by default it will run an accumulation with a few (20) signals
+    // to a set of 4 buckets with Signal To Start
+    triggerContinueAsNewWarning := false;
```

Comment:
Go does not use semicolons (usually)

Reply:
ah, oops, copy/pasted/updated badly. fixed.